### DISCUSSION TITLE
stsdas and table installation

### DISCUSSION THREAD

**Discussion by azim7091** [iraf-community/iraf#141] [Q&A] [answered]
Dear All, 
I have installed IRAF on Fedora (https://iraf-community.github.io/install). But there is not stsdas and table package in it. How can I install stsdas and table packages ?

Thanks in advance
Best Regards
Azim

**Comment by olebole** [comment, marked as answer]
The `tables.ttools` package was integrated into IRAF since version 2.15.1 as `utilities/nttools`. The `stsdas` package and the rest of `tables` however are not part of IRAF and were separately maintained by STScI and are abandoned since 2018. They are not maintained by the iraf-community, since the sources are not available, and they contain code from the "Numerical Recipes" book which is forbidden to distribute.

**Reply by olebole** [reply]
Update: The source (without the files that contain code from the "Numerical Recipes" book) is [here](https://github.com/iraf-community/stsdas). Aside from the NR license problem, the packages are 32-bit only.
